src/components/pages/gallery/Upload.tsx

```diff
@@ -121,11 +156,29 @@ export default function Upload(props: Props) {
         setProgressView(true);
     };
 
+    const resumeDesktopUpload = async (files, collectionName) => {
+        try {
+            pendingDesktopUploadCollectionName.current = collectionName;
+            props.setElectronFiles(files);
+        } catch (e) {
+            logError(e, 'Failed to get previously failed files');
+        }
+    };
+
     function analyseUploadFiles(): AnalysisResult {
-        if (props.acceptedFiles.length === 0) {
+        if (toUploadFiles.current.length === 0) {
             return null;
         }
-        const paths: string[] = props.acceptedFiles.map((file) => file['path']);
+        if (isElectron() && !isUploadDirs) {
```

Comment:
Can we just change the default value of `isUploadDirs` here to `true` ? We can make it to `true` at the end of every upload and it can become false if the user uploads via the file button, maybe can also give it a better name too. Would this work?

Reply:
I didn't understand how that will solve the problem 
To clarify-> I meant the case where the user drags and drops a file instead of using the upload button to upload the file

Reply:
Sorry, I was wrong - I assumed that drag and drop will have relative file paths like on the web and let that analyze function work as it does on web. But since Electron itself adds file paths to every file, it will lead to absolute file paths too.